radiation/python/radlw/radlw_main.py

```diff
@@ -255,7 +255,7 @@ def run(self, plyr, plvl, tlyr, tlvl, qlyr, olyr, gasvmr,
             for i in range(npts):
                 ipseed[i] = icseed[i]
 
-        for iplon in range(npts-1, npts):
+        for iplon in range(0, 1):
```

Comment:
what's this change about, did you flip the vertical indices?

Reply:
This was for my own testing, so that it only ran for the single horizontal point I was comparing it to. Will change back to full range.